src/MonitorProfiler/ProbeInstrumentation/ProbeInstrumentation.cpp

```diff
@@ -189,7 +195,8 @@ HRESULT ProbeInstrumentation::InstallProbes(vector<UNPROCESSED_INSTRUMENTATION_R
     vector<ModuleID> requestedModuleIds;
```

Comment:
I know that this is stack allocated, but the ctors may heap allocated? Do we need to guard OOM from those too?